products/libllbuild/include/llbuild/llbuild.h

```diff
@@ -47,7 +47,9 @@
 /// 1: Added `environment` parameter to llb_buildsystem_invocation_t.
 ///
 /// 0: Pre-history
```

Comment:
I think we should probably move this comment over to the xcconfig as well.

Reply:
I wasn’t sure the best place. The xcconfig seems logical, but isn’t installed with the headers.  We are shipping the xcconfig with the framework, though, so maybe its fine.

Reply:
Hmmm... tough question. Let's leave the authoritative source as the code (easily found by jump-to-definition), and reference this comment in the xcconfig.